css/properties/border-block-style.json

```diff
@@ -69,17 +69,29 @@
                 ]
               }
             ],
-            "opera_android": {
-              "version_added": "48"
-            },
+            "opera_android": [
+              {
```

Comment:
There's no reason for this first opera_android support statement everywhere. I need clean that up